actions/setup-crib-environment/scripts/parse_profiles.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+profiles=$PROFILES
+
+if [ -n "$profiles" ]; then
+  # Convert comma-separated string to array
+  IFS=',' read -r -a profiles_array <<< "$profiles"
+
+  # Initialize result string
+  result=""
+
+  # Loop through each profile in the array
+  for profile in "${profiles_array[@]}"; do
+    result+="-p $profile "
+  done
+
+  # Remove trailing space
+  result=$(echo $result | sed 's/ *$//')
+
+  # Output the result
+  echo "Profiles args: $result"
+  echo "::set-output name=profile_args::$result"
+else
+  echo "No profiles supplied"
+  echo "::set-output name=profile_args::"
```

Comment:
GitHub Actions is trying to deprecate setting outputs like this (see: [this example instead](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/defining-outputs-for-jobs#example-defining-outputs-for-a-job))
```suggestion
  echo "profile_args=$result" | tee -a "$GITHUB_OUTPUT"
else
  echo "No profiles supplied"
  echo "profile_args=" | tee -a "$GITHUB_OUTPUT"
```

Not sure if `echo "profile_args=" | tee -a "$GITHUB_OUTPUT"` is valid -- if not, we can consider just not outputting anything (probably better).